xontrib/prompt_ret_code.xsh

```diff
@@ -2,20 +2,31 @@ from xonsh.tools import ON_WINDOWS as _ON_WINDOWS
 
 
 def _ret_code_color():
+    try:
+	color_bg = $PROMPT_RET_CODE_BACKGROUND
```

Comment:
For xonsh env-vars, we try to start them with `XONSH_` in an effort to namespace them and keep things a bit organized -- this is a xontrib, although a bundled xontrib.  What do you think about prefixing it with `XONTRIB_`?

Reply:
Seems reasonable.